website/docs/docusaurus-core.md

```diff
@@ -113,6 +113,18 @@ function Home() {
 }
 ```
 
+### `<BrowserOnly/>`
+
+This component excludes its `children` during the prerendering stage of the build process. This is useful for hiding code that is only meant to run in the browsers (e.g. where the `window`/`document` objects are being accessed).
+
+```jsx {1,7}
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+<BrowserOnly>
+  {/* Something that should be excluded during build process prerendering. */}
```

Comment:
We need to give an example to show that `children` should be a render function and not normal children.

Reply:
Ah yes, done :heavy_check_mark: